paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -150,6 +150,8 @@ internal class PaymentSheetViewModel @Inject internal constructor(
 
     private var googlePayPaymentMethodLauncher: GooglePayPaymentMethodLauncher? = null
 
+    private var deferredIntentConfirmationType: IntentConfirmationType? = null
```

Comment:
Need to store this here because we need that value after coming back from `PaymentLauncher`.